frontend/src/templates/LandingPage.html

```diff
@@ -5,19 +5,7 @@ <h4>Landing page</h4>
         This promotion has a landing page
     </md-checkbox>
 
-    <div layout="column" ng-show="promotion.landingPage !== undefined" ng-if="newspaper">
-        <md-input-container>
-            <label>Description</label>
-            <textarea name="landing-description" ng-model="promotion.landingPage.description" delete-empty></textarea>
-        </md-input-container>
-        <h5>Default product</h5>
-        <md-radio-group layout="row" ng-model="promotion.landingPage.defaultProduct">
-            <md-radio-button value="voucher" class="md-primary" ng-checked="true">Voucher</md-radio-button>
-            <md-radio-button value="delivery">Home Delivery</md-radio-button>
-        </md-radio-group>
```

Comment:
this was an accident and shouldn't have been removed (although it is unused in the support site and could have been removed). it is needed by the membership-common deserialisers for adding a new promo code (as opposed to editing an existing one)
cc @rupertbates